core/src/main/java/org/bitcoinj/core/NetworkParameters.java

```diff
@@ -726,4 +726,11 @@ public int getDIP0008BlockHeight() {
     public int getBIP65Height() {
         return BIP65Height;
     }
+
+    // Coin Type
+    protected int coinType;
+
+    public int getCoinType() {
+        return coinType;
+    }
```

Comment:
coinType is 5 for mainnet, 1 for all other testnet and devnets.  This will allow the DerivationPathFactory generate the correct path based on the network.